nohost/src/NewStaticServer.js

```diff
@@ -0,0 +1,36 @@
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, regexp: true, indent: 4, maxerr: 50 */
+/*global define, brackets */
+define(function (require, exports, module) {
+    "use strict";
+
+    var BaseServer              = brackets.getModule("LiveDevelopment/Servers/BaseServer").BaseServer,
+        BlobUtils               = brackets.getModule("filesystem/impls/filer/BlobUtils"),
+        Filer                   = brackets.getModule("filesystem/impls/filer/BracketsFiler");
+
+    function NewStaticServer(config) {
```

Comment:
What does the `New` in `NewStaticServer` refer to?  Can we just call this `StaticServer`?